var/spack/repos/builtin/packages/intel-oneapi-dal/package.py

```diff
@@ -29,6 +29,12 @@ class IntelOneapiDal(IntelOneApiLibraryPackage):
     )
 
     if platform.system() == "Linux":
+        version(
+            "2021.7",
```

Comment:
package number is 2021.7.0 in the download URL?

Reply:
Yes, it was RTMed